src/controller.cpp

```diff
@@ -1059,3 +1059,15 @@ bool Controller::deactivate() {
   m_impl->deactivate(stateToReason(m_state));
   return true;
 }
+
+void Controller::forceDaemonSilentServerSwitch() {
+#ifdef MZ_IOS
+  if (m_impl) {
+    logger.debug() << "Sending server switch message to iOS Network Extension";
+    m_impl->forceDaemonSilentServerSwitch();
+  }
+#else
+  logger.debug() << "Server switch debug feature only available for iOS. Not "
```

Comment:
Why make this API even available in platforms other than iOS?

Reply:
I couldn't find a way, but I may have missed something. We need to trigger this from the `ViewDeveloperMenu` QML screen. We have a connection to the controller class, but not any individual platform implementations. (Adding a connection just to iOS's controller seemed heavy handed.) And then within the controller class, we only can call functions on the abstract class. (We can't cast the `m_impl` to `ioscontroller`, for example.)

I'm very open to suggestions for better ways to handle this, because I agree this is pretty messy, especially for a debug option needed just for testing.

Reply:
We can wrap the header declaration in ifdef MZ_IOS. But no hard sell, this is just a nit.

Reply:
Ah sorry, I thought you were going for something otherwise - in my opinion, it's an ugly amount and type of code for a one-platform debug option. I might keep it as is for now - in case we want to add this into Android (for example) in the future.